gcc/common/config/riscv/riscv-common.cc

```diff
@@ -211,6 +212,7 @@ static const struct riscv_ext_version riscv_ext_version_table[] =
 
   {"xcorev",    ISA_SPEC_CLASS_NONE, 1, 0},
```

Comment:
Beyond the scope here, but `xcorev` as an overall ISA extension name covering all extensions is now dropped.  See the  [agreed specification](https://github.com/openhwgroup/core-v-sw/blob/master/specifications/corev-isa-extension-naming.md).

Reply:
See earlier comment. This will be resolved for all of CORE-V gcc and binutils-gdb in a separate PR.